testing/broker_tester_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/pivotal-cf/brokerapi"
 )
 
-//go:generate counterfeiter -o fakes/fake_http_handler.go net/http.Handler
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 -o fakes/fake_http_handler.go net/http.Handler
```

Comment:
👀  this seems like a nice pattern.  Is go mod smart enough not to remove counterfeiter on `go mod tidy` if it appears here?

Reply:
go mod is using the file in `tools/tool.go` to pick up the dep

Reply:
which I just relised @Krenair hasn't committed :raising_hand:

Reply:
done, sorry